src/app/views/pages/_sessions.html.erb

```diff
@@ -1,4 +1,4 @@
-  <%= cache Session.maximum(:updated_at), expires_in: 20.seconds do %>
+  <%= cache [Session.maximum(:updated_at), current_participant], expires_in: 20.seconds do %>
```

Comment:
Would you add a comment here about why we're using current participant as part of the cache key